lib/Runtime/Library/InJavascript/Intl.js

```diff
@@ -552,54 +561,86 @@
             `|${ALPHA}{4,8}`          +                                         //              ~/ 4*8ALPHA            ; (paraphrased: combined previous two lines)
             ')\\b';
 
-        // below: ${language}, ${script}, and ${region} are wrapped in parens because matching groups are useful for replacement
+        // Use matching groups only when needed
         const LANG_TAG_BASE = `\\b(${language})\\b`         +                   // langtag       = language
                               `\\b(?:-(${script}))?\\b`     +                   //                 ["-" script]
                               `\\b(?:-(${region}))?\\b`     ;                   //                 ["-" region]
-        const LANG_TAG_EXT  = `\\b(?:-${variant})*\\b`      +                   //                 *("-" variant)
+        const LANG_TAG_EXT  = `\\b((?:-${variant})*)\\b`    +                   //                 *("-" variant)
```

Comment:
>((?: [](start = 34, length = 4)

Why add another capturing group? Just remove the `?:` to turn the original non-capturing group into a capturing group.